app/assets/javascripts/index/element.js

```diff
@@ -5,7 +5,7 @@
   const isOfExpectedLanguage = ({ language }) => languagesToRequest[0].startsWith(language) || language === "mul";
 
   $(document).on("click", "a[href='#versions-navigation-active-page-item']", function (e) {
-    scrollToActiveVersion();
+    $(document).trigger("numbered_pagination:focus");
```

Comment:
Should that be `:center` as there doesn't seem to be a `:focus` listener?

Reply:
Yes. I renamed it, but this one got lost in an intermediate version.

Updated.